.github/workflows/snapshots.yml

```diff
@@ -32,12 +32,12 @@ jobs:
       - name: Copy Front-End
         run: cp -a ./Public/* ./Tests/AppTests/__Snapshots__/WebpageSnapshotTests
       - name: Diff for HTML Changes
-        id: diff-includes-html-files
+        id: front-end-changes
         run: |
           filecount=$(git diff origin/main --shortstat "**.html" "**.css" "**.js" | awk '{print $1}')
```

Comment:
major: Does this need to be .scss? We don't commit the CSS files into the branch so I don't think `git diff` will ever spot those changes?

Reply:
Really well spotted! 😅